backend/core/api/token/dtos.go

```diff
@@ -1,6 +1,6 @@
 package token
 
 type CreateTokenDTO struct {
-	UserID          uint `json:"userId"`
-	CreatedByUserId uint `json:"createdByUserId"`
+	UserEmail       string `json:"userEmail"`
```

Comment:
why the switch to emails?

Reply:
On the frontend we can't rely on having the correct userID because the user might have just signed up or something. Plus, it is not really good for the admin to have access to the userID. It's just an extra step/API call. If the admin can just enter the invitee's email to match to the account, that is much more reliable than an ID.

Reply:
cool 👍